python/lsst/ts/dimm/dimm_csc.py

```diff
@@ -171,15 +171,10 @@ async def configure(self, config):
         settings = types.SimpleNamespace(**instance)
         controller_class = available_controllers[settings.controller]
         self.controller = controller_class(
-            log=self.log, simulate=self.simulation_mode != 0
+            domain=self.domain, log=self.log, simulate=self.simulation_mode != 0
```

Comment:
I don't like this change. Now all controllers get a domain instance, even the ones that don't need it. Why not simply instantiate the domain instance in the astelco controller and handle everything there?